org.eclipse.wildwebdeveloper.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.eclipse.wildwebdeveloper.feature"
       label="%name"
-      version="0.2.0.qualifier"
+      version="0.2.1.qualifier"
```

Comment:
Since it's a relatively noticeable addition, we'll bump to 0.3.0 to showcase there is more than maintenance fix.